calib.sh

```diff
@@ -13,13 +13,15 @@ outdoor_sync_bag_name=(
 indoor_sync_bag_name=(
 #"2021-12-20-11-34-55_0.bag"
 #"2021-12-22-12-29-01.bag"
-"2021-12-22-12-15-09.bag"
+#"2021-12-22-12-15-09-fixed.bag"
 #"Garage-01.bag"
 #"Garage-03.bag"
 #"Garage-04.bag"
 #"Garage-05.bag"
-#"test-300-garage-fixed.bag"
-#"test600garage-fixed.bag"
+#"test-300-garage-fixed-combined.bag"
+#"test600garage-fixed-combined.bag"
+#"calib_01-fixed-combined.bag"
+"22_1_13/2022_1_13_calib_4.bag"
 )
```

Comment:
I would remove all these .bag files and leave just one (for example)

Reply:
agreed

Reply:
Did that and removed couple of other lines too